mir_eval/separation.py

```diff
@@ -328,7 +438,8 @@ def _safe_db(num, den):
     return 10 * np.log10(num / den)
 
 
-def evaluate(reference_sources, estimated_sources, **kwargs):
+def evaluate(reference_sources, estimated_sources,
+             framewise=False, window=30*44100, hop=15*44100, **kwargs):
```

Comment:
1. Ahh! Sorry, I was confused about where the default should be defined.. I'll make that change now.

2. Okay, I'll have it call both. I'll do a bit of reworking on the framewise function to incorporate the feedback from @aliutkus vis-a-vis handling short frames.